packages/office-ui-fabric-react/src/components/Persona/Persona.base.tsx

```diff
@@ -114,32 +120,48 @@ export class PersonaBase extends BaseComponent<IPersonaProps, {}> {
     );
   }
 
+  /**
+   * to render various types of Text based on the className
+   * primaryText, secondaryText, tertinaryText, and optionalText
```

Comment:
>primaryText, secondaryText, tertinaryText, and optionalText [](start = 5, length = 59)

There is no strict dependency on only using these classnames: the function can take in any kind of classname and render with that